tests/RockyConstructor.agent.test.nut

```diff
@@ -38,37 +38,82 @@ class RockyConstructor extends Core {
         values = [null, true, 0, -1, 1, 13.37, "String", [1, 2], {"counter": "this"}, blob(64), function(){}];
     }
 
-    function testRockyAccessControlOption() {
```

Comment:
why do we need these test updates?

Reply:
That group of tests (`testRocky[SOMETHING]Option()`) wasn't actually changed in this update, not sure how it ended up included. So no, not needed. 

`testMultipleAsyncRequestsWithSendToAll()` and `testContentTypeMultipart()` were what were actually added in this update.

Reply:
If these are not supposed to be added/changed, let's please just revert it. I still see them in the PR as the changes to be merged.